### ISSUE TITLE
create-nx-workspace - Fails when there's a space in the path - Windows

### ISSUE THREAD

**Issue by PieterjanDeClippel** [nrwl/nx#9282]
## Current Behavior

![image](https://user-images.githubusercontent.com/9629574/157822176-6583d8f2-29a5-4808-bc98-c2a96f54c962.png)

`create-nx-workspace` fails due to a space in the path

## Expected Behavior

Somewhere in the code the path should be wrapped with quotes

### Environment

OS: Windows 11 Pro - Build 22000.556

```
Angular CLI: 13.2.6
Node: 16.13.1
Package Manager: npm 8.3.2
OS: win32 x64

Angular: undefined
...

Package                      Version
------------------------------------------------------
@angular-devkit/architect    0.1302.6 (cli-only)
@angular-devkit/core         13.2.3
@angular-devkit/schematics   13.2.3
@schematics/angular          13.2.3
```

**Comment by PieterjanDeClippel** [comment]
Man your pull-request flow is an absolute real horror on Windows machines!!!

- The repository must not be cloned in a path containing spaces 😑 because it causes the git hook to fail
- I followed [these steps](https://github.com/nrwl/nx/blob/master/CONTRIBUTING.md#-submitting-a-pr)
- When trying to push my commit to my new branch in my fork on Github, I'm getting an error that the push fails, and VSCode suggests me to pull first (which absolutely doesn't make sense, but I decided to do this anyway). This gets all the readme's in modified state in my repository (without anything actually being changed).

![image](https://user-images.githubusercontent.com/9629574/159228432-52183b00-786c-4996-8a8f-28ad9fe6e3ac.png)

When hitting "Show log", I see the following

![image](https://user-images.githubusercontent.com/9629574/159229130-dd9cd598-0c8f-4d5d-92c7-282e3de6d460.png)

I've done tons of pull-requests in the past, and never experienced anything like this.

The only change that needs to be done to fix this issue is [here](https://github.com/nrwl/nx/blob/master/packages/create-nx-workspace/bin/create-nx-workspace.ts#L547):

    const fullCommand = `${fullCommandWithoutWorkspaceRoot} --nxWorkspaceRoot="${nxWorkspaceRoot}"`;

Thanks for doing this for me, and not requiring me to create a pull-request
